tests/chainerx_tests/unit_tests/routines_tests/test_misc.py

```diff
@@ -125,22 +125,30 @@ def forward_expected(self, inputs):
         'in_dtypes,out_dtype': math_utils.in_out_float_dtypes_math_functions,
         'input': ['random'],
         'contiguous': [None, 'C'],
+        'is_module': [True, False],
```

Comment:
This use of `is_module` parameter is confusing, because it's different from other test cases.
How about using different name or value, such as `is_buitin=True/False` or `is_module=True/'builtin'`?

Reply:
Yes I thought of that too. This is on the other hand consistent with https://github.com/chainer/chainer/blob/master/tests/chainerx_tests/unit_tests/routines_tests/test_math.py#L146-L152.

Reply:
Hmm. I see.